_i18n/es/resources/user-guides/monerod-as-a-systemd-service.md

```diff
@@ -0,0 +1,73 @@
+{% include untranslated.html %}
+# Using monerod as a systemd Service
+
+If you are using a Linux distribution that uses `systemd`, you can use it's
+user units to manage `monerod` as well.
+
+We will do this in the command line & assume that you have already built or
+downloaded `monerod`.
+
+
+## Create the Service File
+
+First start by creating the directory for your user service:
+
+    mkdir -p ~/.config/systemd/user/
```

Comment:
Do you think that having it as a system unit and a having separate user and group to run monerod would be better?

Reply:
Yeah that's definitely the better option, I just wasn't sure how complex I could/should get. User service seemed simpler & doesn't require `sudo`.

I'm down to change the main guide to creating & using a separate user/group & add just adding a section at the end for using user units if you don't have admin permissions.